ukbb_qc/assessment/sanity_checks.py

```diff
@@ -367,9 +367,13 @@ def sample_sum_sanity_checks(
     # Check if pops are present
     # Get list of all pops present in HT
     pop_adjusted = list(
-        set([x for x in info_metrics if (("adj" in x) and ("raw" not in x))])
+        # This pulls out fields in the info metric that look something like AC_<pop>_adj
+        # It splits on the underscore and takes the second field to extract the pop label
+        # Note that this will also extract the strings 'adj', 'XX', and 'XY'
+        # because the info metrics will contain fields for AC_adj, AC_XX_adj, and AC_XY_adj
+        set([x.split("_")[1] for x in info_metrics if (("adj" in x) and ("AC" in x))])
     )
-    pop_adjusted = set([i.replace("_adj", "") for i in pop_adjusted])
+    pop_adjusted = set(x for x in pop_adjusted if x not in ("XX", "XY", "adj"))
```

Comment:
this function is a bit of a mess, but:

- lines 367-376 pull pop labels out of the info metrics
- lines 378-384 pulls pop labels out of the freq meta and compares them to the UKBB pops constant
- lines 386-392 compares the pops found in the freq meta to the pops found in the info metrics

if the second section errors out (instead of just displaying a warning), then this function will compare the pops in the info metrics and the freq meta to the UKBB pops constant. I will add that now but want to note this step has already been run